src/main/app/case/definition.ts

```diff
@@ -2420,3 +2420,25 @@ export interface C1ASafteyConcerns {
       statement: string;
     };
   };
+
+  export type ChildrenDetailsTranslations = {
```

Comment:
@giorgi-hmcts Why we need this ?
We already have child details shape defined 'ChildrenDetails'

Reply:
Removed the redundant `ChildrenDetailsTranslations` from `definition.ts`